compiler/modules/types/mapping.bal

```diff
@@ -422,19 +426,18 @@ function mappingAtomicMemberType(MappingAtomicType atomic, string? key) returns
     return m;
 }
 
-function bddMappingMemberRequired(Context cx, Bdd b, string k, boolean requiredOnPath) returns boolean {
+function bddMappingMemberRequired(Context cx, Bdd b, StringSubtype k, boolean requiredOnPath) returns boolean {
     if b is boolean {
         return b ? requiredOnPath : true;
     }
     else {
         return bddMappingMemberRequired(cx, b.left, k,
-                                        requiredOnPath || cx.mappingAtomType(b.atom).names.indexOf(k) != ())
+                                        requiredOnPath || stringSubtypeFindIn(k, cx.mappingAtomType(b.atom).names)[1])
```

Comment:
@jclark should we extract a variable here? Not sure if it's ok just get the second element from the tuple.